user snippets/networkCalls.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Run a small snippet of JavaScript during a mabl flow/journey
+ * 
+ * @param {object} mablInputs - Object containing mabl inputs such as variables (mablInputs.variables). 
+ *                              Use mablInputs.variables.user for user defined variables
+ *                              (For example myVar may be accessed as mablInputs.variables.user.myVar)
+ * 
+ * @param {function} callback - A callback function that must be called to complete 
+ *                              the javascript step and provide a value to the following 
+ *                              steps of the flow/journey. A return statement from this
+ *                              function call will not provide any results for use
+ *                              in the following steps in this flow or journey.
+ */
+
+function mablJavaScriptStep(mablInputs, callback) {
+  // document.defaultView returns the window object in your browser
+  // document.defaultView.performance.getEntries() returns a list of all network calls made (works in latest Chrome and Firefox)
+  const networkCalls = document.defaultView.performance.getEntries();
```

Comment:
^ For extra safety this should be `var`. This code runs directly in the browser, not through babel, so `const` will break IE11

Reply:
love it added